.dockerignore

```diff
@@ -6,6 +6,5 @@ __pycache__
 **/_templates
 **/build
 **/.vscode
-**/.docker
```

Comment:
The isaac tutorial has a `.docker` folder and this was keeping it from building properly.